data/json/recipes/other/tool.json

```diff
@@ -90,6 +90,28 @@
     ]
   },
   {
+    "result": "pickaxe",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 7,
```

Comment:
If it is dropped a level, then it would be pointless to have it learned from `textbook_fabrication` at level 6, unless I should set it back to not being autolearned?